esp/esdllib/README.md

```diff
@@ -0,0 +1,2142 @@
+# ESDL Scripting
+
+ESDL started as a way to standardize and streamline service and microservice development, and has seen continual improvement to support more complex uses. Its foundational features let us stand up a front end service for a roxie backend with just an interface description and no other customized logic or coding. Starting there we have the benefit of robust interfaces isolated from the compiled code of the HPCC Platform itself. You can easily update a service's interface or its configuration binding it to the backend without building or deploying any new C++ code.
```

Comment:
@asselitx Maybe we should put a "## History" title above this text and add a paragraph above simply describing how powerful ESDL scripting is in it's current state?